### PR TITLE
Publishing crates on release

### PR DESCRIPTION
**Pull request by mutantcornholio**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [workspace.package]
-version = "0.0.8"
+version = "0.0.0-updated-from-ci"
```
Comment (ggwpez) [inline Cargo.toml:2]: Is this just as a test?

Comment (mutantcornholio) [reply Cargo.toml:2]: No, the idea is, to set the version from CI automatically. This way, we'll only need to "release" the package in GitHub, without having to also bump the version in Cargo.toml

Comment (ggwpez) [reply Cargo.toml:2]: Okay, and the modified version is the pushed again to a branch or tag to download?

**Review by ggwpez — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (mutantcornholio) [reply Cargo.toml:2]: It doesn't. It just gets pushed to creates.io.  
Do you see any downsides to this approach? I don't have much experience using it with cargo, but we've been doing that for a while for npm packages, and it's been good so far.

**Review by EgorPopelyaev — APPROVED** [review]

**Pull request merged**